src/main/java/commands/AddCommand.java

```diff
@@ -0,0 +1,27 @@
+package commands;
```

Comment:
Would it be better if the root name of the package was the project name?
```suggestion
package duke.commands;
```

Reply:
Absolutely, great catch again!